tests/Assembly/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -125,6 +125,7 @@ loop:
     a add swap1
     n := sub(n, 1)
     jump(loop)
+
```

Comment:
Why does this have a space but the `one:`, `two:`, etc. labels below don't?

Reply:
yeah, they actually got removed

Reply:
I added a space here just to have the same code as the example in #449
That way we have an example of no empty line before and 1 empty line before

Reply:
> yeah, they actually got removed

yeah, before this PR we were always printing an extra line. In the other snapshots, those lines got removed in the output.
This line is from the input of `tests/Assembly/Assembly.sol` so it is added.

Reply:
Oh, sorry :man_facepalming: I shouldn't review code when I'm too tired.